scapy/config.py

```diff
@@ -345,6 +345,7 @@ class Conf(ConfClass):
     L3socket = None
     L2socket = None
     L2listen = None
+    min_pkt_size = 60
```

Comment:
Because that's not the case for every driver, I have one example where 58 bytes are needed.

I don't think than more than 60 could be needed, since that is the minimum ethernet length, and I think it's fine to have the value set to 60 by default as it can be easily configured.

Also, I have not found a way to find this value dynamically (other than by bisecting, and I don't think that trying multiple values is a good option).